src/main/java/com/example/Main.java

```diff
@@ -1,7 +1,21 @@
 package com.example;
 
+/**
+ * Main class that contains void main
+ *
+ * @author Elena Yevsyukova
+ * @version 1.1
+ */
 public class Main {
+    /**
+     * Here starts point of the program
```

Comment:
This is not correct from the English point of view. The sentence should finish with period. Again, it tells what a programmer already knows about  the `main()` method.

What does the method do? Why are we having it?